inductiva/_cli/cmd_logs/logs.py

```diff
@@ -1,18 +1,35 @@
-"""Consume the stream of logs of a task."""
+"""CLI for logs."""
+import sys
 
-from inductiva.tasks import streams
+from .. import utils as cli_utils
+from ... import tasks
```

Comment:
Why the relative import outside an `__init__.py` file?. Why no just `from inductiva import tasks`. I am under the impression that this is the preferred method according to the google style guides